textattack/loggers/attack_log_manager.py

```diff
@@ -20,14 +20,17 @@ def __init__(self):
     def enable_stdout(self):
         self.loggers.append(FileLogger(stdout=True))
 
-    def enable_visdom(self, **kwargs):
-        self.loggers.append(VisdomLogger(**kwargs))
+    def enable_visdom(self):
+        self.loggers.append(VisdomLogger())
 
-    def enable_wandb(self, *args):
-        self.loggers.append(WeightsAndBiasesLogger(*args))
+    def enable_wandb(self):
+        self.loggers.append(WeightsAndBiasesLogger())
 
-    def add_output_file(self, filename):
-        self.loggers.append(FileLogger(filename=filename))
+    def disable_color(self):
+        self.loggers.append(FileLogger(stdout=True, color_method="file"))
+
+    def add_output_file(self, filename, color_method):
+        self.loggers.append(FileLogger(filename=filename, color_method=color_method))
```

Comment:
Will the `color_method` ever not be `"file"` here?

Reply:
Yes, when the user decides not to redirect stdout to a file the color
method will be set to ansi.

Reply:
No, I get that. However, in this `add_output_file` function, the `color_method` will always be `"file"` I think. 
Can you just change to `color_method = "file"` in L33?